test3.tf

```diff
@@ -0,0 +1,131 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.0"
+    }
+  }
+}
+provider "azurerm" {
+  features {}
+}
+
+resource "random_integer" "rnd_int" {
+  min     = 1
+  max     = 10000
+}
+
+resource random_string "password" {
+  length      = 16
+  special     = false
+  min_lower   = 1
+  min_numeric = 1
+  min_upper   = 1
+}
+
+resource "azurerm_sql_firewall_rule" "example" {
+  name                = "firewall-rule"
+  resource_group_name = "testrg"
+  server_name         = azurerm_sql_server.example.name
+  start_ip_address    = "10.0.17.62"
+  end_ip_address      = "10.0.17.62"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "sa${random_integer.rnd_int.result}"
+  resource_group_name      = "testrg"
+  location                 = "East US"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+  queue_properties {
+    logging {
+      delete                = false
+      read                  = false
+      write                 = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+    hour_metrics {
+      enabled               = true
+      include_apis          = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+    minute_metrics {
+      enabled               = true
+      include_apis          = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+  }
+}
+
+resource "azurerm_sql_server" "example" {
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that Azure Active Directory Admin is configured'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- a3967aca-66c9-4aa9-af7a-af14fab61dc4 -->